WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -187,7 +190,8 @@ class CommentDetailViewController: UITableViewController {
         self.isLastInList = isLastInList
         self.managedObjectContext = managedObjectContext
         self.replyID = comment.replyID
-        super.init(style: .plain)
+        super.init(nibName: nil, bundle: nil)
+        configureView()
```

Comment:
Not a major issue, but calling this method in `init` causes the `viewDidLoad` to be prematurely called (since `configureView` calls the `view` property which triggers `viewDidLoad`) – this caused the `navigationController` to be nil while configuring the navigation bar, thus causing `translucent` property to stay false.

This caused a minor visual bug where the content won't scroll below the navigation bar (since it's not translucent). What do you think about moving this to `viewDidLoad`? I've tried it, and it seems to fix the issue. 🙂

Reply:
Nice catch! Will do!